tests/output/ConsoleInteractionTest.py

```diff
@@ -151,23 +151,27 @@ def tearDown(self):
     def test_require_settings(self):
         self.assertRaises(TypeError, acquire_settings, self.log_printer, 0)
 
+        curr_section = Section("")
```

Comment:
I read through the code and I don't believe it should be in `test_fail_acquire_settings` there is the output for when an acquired setting is bad